cls/SourceControl/Git/Utils.cls

```diff
@@ -1138,10 +1138,12 @@ ClassMethod ImportRoutines(force As %Boolean = 0) As %Status
     for  {
         set internalName = $order(itemList(internalName))
         quit:internalName=""
-        if '..IsInSourceControl(internalName) {
-            continue
+        #dim sc as %Status
+        if ..IsInSourceControl(internalName) {
+            set sc = ..ImportItem(internalName, force)            
+        } else {
+            set sc = ..AddToServerSideSourceControl(internalName)
```

Comment:
Are such newly added files actually imported? Looking at the diff I'd think not, but maybe this happens deeper down - haven't actually tested. Presumably we'd always want to call ImportItem?

Reply:
Yes, `AddToServerSideSourceControl` first updates the global and than calls `ImportItem`.

```Objectscript
ClassMethod AddToServerSideSourceControl(InternalName As %String) As %Status
{
     #dim i as %Integer
    #dim ec as %Status = $$$OK
    for i = 1:1:$length(InternalName, ",") {
        #dim item as %String = ..NormalizeExtension($piece(InternalName, ",", i))
        set @..#Storage@("items", item) = ""
        #dim sc as %Status =  ..ImportItem(item, 1)
        if 'sc {
            set ec = $$$ADDSC(ec, sc)
        }
    }
    quit ec
}
```